src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -16,28 +16,108 @@ public class Line extends BaseEntity {
     private String name;
     private String color;
 
-    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
-    private List<Section> sections = new ArrayList<>();
+    @Embedded
+    private final Sections sections;
 
     public Line() {
+        this.sections = new Sections();
     }
 
     public Line(String name, String color) {
         this.name = name;
         this.color = color;
+        this.sections = new Sections();
     }
 
     public Line(String name, String color, Station upStation, Station downStation, int distance) {
         this.name = name;
         this.color = color;
-        sections.add(new Section(this, upStation, downStation, distance));
+        this.sections = new Sections();
+        this.sections.addSection(new Section(this, upStation, downStation, distance));
```

Comment:
```suggestion
        addSection(upStation, downStation, distance);
```

추후 유지보수성을 고려한다면 동일한 메서드를 사용하는게 좋을 것 같은데 어떻게 생각하시나요?

Reply:
한곳으로 통일하는 방향으로 수정하겠습니다!